bridge/client/app/_components/ktb-keptn-services-list/ktb-keptn-services-list.component.ts

```diff
@@ -67,10 +71,18 @@ export class KtbKeptnServicesListComponent implements OnInit, OnDestroy {
 
   ngOnDestroy(): void {
     this.unsubscribe$.next();
+    this.unsubscribe$.complete();
   }
 
   public setSelectedUniformRegistration(uniformRegistration: UniformRegistration) {
     if (this.selectedUniformRegistration !== uniformRegistration) {
+      this.lastSeen = this.dataService.getUniformDate(uniformRegistration.id);
+      if (this.selectedUniformRegistration) {
+        this.selectedUniformRegistration.unreadEvents = 0;
+        if (this.uniformRegistrations.data.some(registration => registration.unreadEvents !== 0)) {
```

Comment:
Please change to `if (!this.uniformRegistrations.data.some(registration => registration.unreadEvents !== 0))`

Reply:
done